flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonBatchFileSystemITCase.java

```diff
@@ -50,11 +54,27 @@ public String[] formatProperties() {
     }
 
     @Test
-    public void testParseError() throws Exception {
-        String path = new URI(resultPath()).getPath();
+    void testParseError(@TempDir java.nio.file.Path temporaryFolder) throws Exception {
+        String sql =
```

Comment:
where does this come from?

Reply:
it comes from FileSystemITCaseBase.open， 
This is a basic class. I dare not change it without authorization. I can only select the fragments required for code execution.

Reply:
That approach will just create a mess when the FileSystemITCaseBase has been migrated, as no one will have a clue that something was copied into the tests themselves.
If this test doesn't work as expected without the FileSystemITCaseBase etc. being migrated, then this test should still use junit4 (but still use assertj).

Reply:
ok, i'm changing it back to junit4 now, so @RegisterExtension and @TempDir won't work because they need junit5, i'm back to @Rule

Reply:
thanks!